application/dash_application/dash_multipage_index.py

```diff
@@ -28,8 +38,12 @@
 
 
 def init_callbacks(dash_app):
+    ## Needed for multipage support
     dash_app.config.suppress_callback_exceptions = True
 
+    ## Detects change in URL and outputs layout objects to page-content div
+    ## Flask (dash_app.server)'s cache extension holds a map of input & output
+    ## in memory to improve performance. 
```

Comment:
The way this currently works, even URLs that do not have `/dash/` (example: http://127.0.0.1:5000/askljflaksjf/dafkljd) will return the dash landing page. This is a minor issue and does not break anything.